build.gradle

```diff
@@ -45,7 +45,6 @@ subprojects {
 
     dependencies {
         compile "org.eclipse.xtend:org.eclipse.xtend.lib:$versions.xtext"
-        // use newer Guice version (4.1 instead of 4.0)
```

Comment:
why this comment?

Reply:
it is removed. since guice is already upgraded to 4.1. why this comment was here, I cannot remember (I think Franz had a problem with outer builds which turned out to be a bug in < 4.1)